src/components/row/column/column.d.ts

```diff
@@ -1,36 +1,20 @@
 import * as React from 'react';
 
 export interface ColumnProps {
-    /**
-     * Classes to apply to the component.
-     */
+    /** Children. */
+    children: React.ReactNode;
```

Comment:
`children` is optional in the propTypes in the component but required here, one or the other needs to change